modules/mobile/app/services/mobile/v0/profile/sync_update_service.rb

```diff
@@ -129,7 +129,7 @@ def check_transaction_status!(transaction_id)
         end
 
         def contact_information_service
-          VAProfile::ContactInformation::Service.new @user
```

Comment:
This should be behind a flipper to ensure  VAProfile::V2::ContactInformation::Service.new @user is not initialized in production. 
```
        def contact_information_service
          if Flipper.enabled?(:mobile_v2_contact_info, @user)
            VAProfile::V2::ContactInformation::Service.new @user
          else
            VAProfile::V2::ContactInformation::Service.new @user
           end
        end
```

Reply:
The VCR response in va_profile/v2/contact_information/put_address_transaction_status and put_address_success do not match the address in the va_profile_v3_address, :v2 factory.
https://github.com/department-of-veterans-affairs/vets-api/blob/3c7151d440d2e6c9e8bee6f3027ddcc7d0df6964/modules/mobile/spec/services/sync_update_service_spec.rb#L26C1-L36C8